icesat2_boreal_stac/constants.py

```diff
@@ -51,3 +47,264 @@ def matches_file(self, filename: str) -> bool:
     def required_assets(cls) -> Set[str]:
         """Returns set of all required asset types"""
         return {member.value for member in cls}
+
+
+VERSION = "v2.1"
+COLLECTION_ID_FORMAT = "icesat2-boreal-{version}-{variable}"
+
+RDS_MEDIA_TYPE = "application/x-rds"
+CSV_MEDIA_TYPE = "text/csv"
+
+RASTER_SIZE = 3000
```

Comment:
This constant is confusing, it's only used as an arg to rio_stac get_raster_info - which then only uses it if it's bigger than the larger of height or width. I think at minimum there needs to be a comment here explaining how/why to set this value. OR to remove it and just let the default of the function operate. https://github.com/developmentseed/rio-stac/blob/219c84f0342b5ac58e3a1a68b43eb50f3f30dda3/rio_stac/stac.py#L181

Reply:
I'll just get rid of it